dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -467,10 +467,16 @@ private void resolve(Object installObject) {
         Collection<List<RepositoryResource>> resolveResult = null;
         RepositoryResolver resolver = null;
         Collection<String> featuresResolved = new ArrayList<String>();
-
+        String edition = null;
         try {
             for (ProductInfo productInfo : ProductInfo.getAllProductInfo().values()) {
                 productDefinitions.add(new ProductInfoProductDefinition(productInfo));
+                edition = productInfo.getEdition();
+            }
+            if (edition.contains("Open")) {
```

Comment:
It would be better to check productInfo.getId() for "io.openliberty" instead of checking the edition string.
Then below, instead of passing in the edition to ExceptionUtils, you could pass in a boolean for whether it is Open Liberty, then let that determine which message to use.